src/filters/filters.ml

```diff
@@ -396,6 +396,32 @@ let check_abstract_as_value e =
 	loop e;
 	e
 
+let check_no_closure_meta e =
+	let error name pos = error ("Field " ^ name ^ " cannot be used as a value") pos in
```

Comment:
Can this meta be applied on a non-method field? Could the error say `"Method " ^ ...`?

Reply:
It can be applied to `var` field, but the check only handles `function` fields. But ok.